operator/doublewrite/metrics.go

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package doublewrite
+
+import (
+	"github.com/cilium/cilium/pkg/metrics"
+	"github.com/cilium/cilium/pkg/metrics/metric"
+)
+
+func NewMetrics() *Metrics {
+	return &Metrics{
+		IdentityCRDTotalCount: metric.NewGauge(metric.GaugeOpts{
+			Namespace: metrics.CiliumOperatorNamespace,
+			Name:      "identity_crd_total_count",
```

Comment:
I wonder if it could make sense to include `doublewrite` somewhere in the name (maybe `identity_doublewrite_crd_total_count`) to explicit the fact that these metrics are output only in doublewrite mode.